ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/activeMapping/StancePoseCalculator.java

```diff
@@ -71,13 +57,14 @@ public void insertCandidatePoses(ArrayList<FramePose3D> poses, FramePose3D goalP
       poses.clear();
```

Comment:
makes more sense to call `populateCandidatePoses` and to rename poses as `posesToPack` so it's clear it's a list getting populated. 

Also, can change goalPose to `FramePose3DReadOnly`

Reply:
Like that too, nice